proto/ibc/core/client/v1/tx.proto

```
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package ibc.core.client.v1;
```

buf (1.47.2)

Files with package "ibc.core.client.v1" must be within a directory "ibc/core/client/v1" relative to root but were in directory "proto/ibc/core/client/v1".

(PACKAGE_DIRECTORY_MATCH)